public/openapi.yaml

```diff
@@ -3,7 +3,7 @@ swagger: "2.0"
 info:
```

Comment:
Tror kanskje ikke denne validerer lenger; kreves en separat modell-definisjon ved flere body-parametere?

64: Semantic error at paths./kpi/{id}.post.parameters - Multiple body parameters are not allowed.
143: Semantic error at paths./kpi/{id}/values/{date}.put.parameters - Multiple body parameters are not allowed.
217: Semantic error at paths./keyres/{id}.post.parameters - Multiple body parameters are not allowed.

Reply:
Ah! Hm, hvor kom de feilmeldingene?

Reply:
https://swagger.io/tools/swagger-editor/

Reply:
Ja, jeg lurte egentlig på om vi hadde validering som en del av byggeprosessen noe sted. :sweat_smile: Hvis ikke burde vi kanskje satt det opp som en del av lintinga, for eksempel?

Reply:
Gjort i https://github.com/oslokommune/okr-tracker/pull/665. ;)